reference.md

```diff
@@ -161,6 +161,10 @@ mutable
 :   Changeable. The value of mutable data can be altered after it has been
     created. See [immutable](#immutable)."
 
+notebook
+:   Interactive computational environment in your web browser, in which you can write and execute Python code.
+    Examples are IPython or Jupyter notebooks
+
```

Comment:
1. To me "in your web browser" is slightly vague. Does it sound any more clear to you to say "Interactive computational environment accessed via your web browser"? 
2. Should we also indicate they can contain text and visualizations? Something like "write and execute Python code plus display text and visualizations"?
3. To follow the style of other glossary entries, I would end the text with a full stop (`.`).

Reply:
Perhaps we could base our definition off the one used by the Jupyter project itself:

_from https://jupyter-notebook.readthedocs.io/en/latest/notebook.html#main-features-of-the-web-application_

> a browser-based tool for interactive authoring of documents which combine explanatory text, mathematics, computations and their rich media output.

Reply:
@ldko @maxim-belkin Thanks for your feedback and suggestions. I tried to combine both of your ideas, maybe you can have second look. As I am not a native speaker, I would trust your recommendations ;-)